docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
 
 .. code-block:: bash
 
-  $ cd streamalert
-  $ pip install --user virtualenv
-  $ virtualenv -p python2.7 venv
-  $ source venv/bin/activate
+  pip install --user virtualenv
+  virtualenv --version
 
-5. Install dependencies:
+4. If you're on a Linux system, you may need to install the Python development libraries:
 
 .. code-block:: bash
 
-  $ pip install -r requirements.txt
+  sudo apt install python-dev    # Debian
+  sudo yum install python-devel  # CentOS/RHEL
 
+Download StreamAlert
+--------------------
 
-.. note:: If you run into issues with ``psutil`` installation, make sure you have the ``python-dev`` dependencies installed.  `debian: sudo apt install python-dev``, `CentOS/RHEL: sudo yum install python-devel``
+1. Clone the latest stable release of StreamAlert:
 
+.. code-block:: bash
+
+  git clone --branch stable https://github.com/airbnb/streamalert.git
+
+2. Create and activate a virtual environment:
+
+.. code-block:: bash
+
+  cd streamalert
+  virtualenv -p python2.7 venv
+  source venv/bin/activate
+
+3. Install the StreamAlert requirements:
+
+.. code-block:: bash
+
+  pip install -r requirements.txt
+
+4. Run unit tests to make sure everything is installed correctly:
+
+.. code-block:: bash
+
+  tests/scripts/unit_tests.sh
+
+Set AWS Credentials
+-------------------
+
+1. Create an AWS account and an IAM user with permissions for at least the following services:
+
+* Athena
+* CloudTrail
+* CloudWatch Events and Logs
+* DyanmoDB
+* IAM
+* Kinesis Firehose and Streams
+* KMS
+* Lambda
+* S3
+* SNS
+* SQS
+
+2. Configure your AWS credentials
+
+.. code-block:: bash
+
+  pip install --user awscli
+  aws configure
+
+Deploy
+------
+
+1. Set basic StreamAlert configuration options:
+
+.. code-block:: bash
+
+  ./manage.py configure aws_account_id 123412341234  # Replace with your 12-digit AWS account ID
+  ./manage.py configure prefix company_team          # Choose a unique name prefix
+
+2. Build the StreamAlert infrastructure:
+
+.. code-block:: bash
+
+  ./manage.py terraform init
+
+There will be multiple Terraform prompts, type "yes" at each one to continue.
+
+3. At this point, StreamAlert is up and running! You can, for example, see the S3 buckets
+that were automatically created:
+
+.. code-block:: bash
+
+  aws s3 ls | grep streamalert
+
+You can also login to the AWS web console and see StreamAlert's CloudWatch logs, Lambda functions, etc.
+
+Live Test
+---------
+Now let's upload some data and trigger an alert to see StreamAlert in action! Our example uses
+SNS for both sending the log data and receiving the alert but StreamAlert also supports many other
+`data sources <datasources.html>`_ and `alert outputs <outputs.html>`_.
+
+1. Create 2 SNS topics:
+
+.. code-block:: bash
+
+  aws sns create-topic --name streamalert-test-data
+  aws sns create-topic --name streamalert-test-alerts
+
+2. Subscribe your email to the alerts SNS topic (replacing REGION, ACCOUNTID, and EMAIL with your own values):
```

Comment:
Suggestion: You could have them export these values into their shell env and then use then env var here (and in other commands), so they could just copy/paste and not reason about replacing values

Reply:
Great idea, will do. This doesn't work when modifying the config files, but would make the bash commands work without modification.